app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/SearchRequestAutoMapperProfile.cs

```diff
@@ -22,7 +22,7 @@ public SearchRequestAutoMapperProfile()
                  .ForMember(dest => dest.EffectiveDate, opt => opt.MapFrom(src => src.IdentificationEffectiveDate))
                  .ForMember(dest => dest.ExpirationDate, opt => opt.MapFrom(src => src.IdentificationExpirationDate))
                  .ForMember(dest => dest.Type, opt => opt.ConvertUsing(new IdentifierTypeConverter(), src => src.IdentifierType))
-                 .ForMember(dest => dest.IssuedBy, opt => opt.MapFrom(src => src.IssuedBy));
+                 .ForMember(dest => dest.IssuedBy, opt => opt.ConvertUsing(new IssueByTypeConverter(), src => src.InformationSource));
```

Comment:
Use the string value instead of the integer here

Reply:
ok